src/script/E2EIdentity/E2EIdentityEnrollment.ts

```diff
@@ -387,17 +387,12 @@ export class E2EIHandler extends TypedEventEmitter<Events> {
     });
   }
 
-  private shouldShowNotification(): boolean {
-    // If the user has already snoozed the notification, don't show it again until the snooze period has expired
-    if (this.currentStep === E2EIHandlerStep.SNOOZE) {
-      return false;
-    }
-    return true;
-  }
```

Comment:
This is actually covered by the check on this line https://github.com/wireapp/wire-webapp/pull/16545/files#diff-080531c7467e54b71bd1e9e26a2d57a487984bf982fefb3f7afc152640f7da23R435